mobile/src/main/java/ch/epfl/sdp/mobile/ui/i18n/LocalizedStrings.kt

```diff
@@ -148,4 +148,9 @@ interface LocalizedStrings {
   val tournamentsCreateDirectElimination: String
   val tournamentsCreateActionCancel: String
   val tournamentsCreateActionCreate: String
+
+  val tournamentsCreateQualifierSize0: String
+  val tournamentsCreateQualifierSizeN: (size: String) -> String
+  val tournamentsCreateElimDemomN: (denominator: String) -> String
```

Comment:
```suggestion
  val tournamentsCreateQualifierSizeN: (size: Int) -> String
  val tournamentsCreateElimDemomN: (denominator: Int) -> String
```

Reply:
Fixed in https://github.com/epfl-SDP/android/pull/336/commits/2c49c24e602fdb287bda38dcf3e6c2d4be32f256